### PR TITLE
Update redeemables.md

### PR DESCRIPTION
**Pull request by louisholley**

### CODE REVIEW

File: src/doc/fxhash/redeemables.md [hunk 1]
```diff
@@ -8,15 +8,15 @@ description: "Redeemables provide a framework to connect NFTs with their physica
 
 # What are redeemables ?
```
Comment (cosimopiu) [inline src/doc/fxhash/redeemables.md:9]: # What are redeemables?

Comment (cosimopiu) [inline src/doc/fxhash/redeemables.md:9]: # What are redeemables?

File: src/doc/fxhash/redeemables.md [hunk 2]
```diff
@@ -8,15 +8,15 @@ description: "Redeemables provide a framework to connect NFTs with their physica
...
 
-Redeemables provide a set of modules allowing the platform to request arbitrary data, which gets signed and is associated to a redeem event when it occurs on-chain. It allows us to map on-chain redemptions with the data signed and collected off-chain, which in turns enables various physical events tooccur, mainly shipping a product associated with a NFT.
+Redeemables provide a set of modules allowing the platform to request arbitrary data. This data is then signed and is associated with a redeem event when it occurs on-chain. It enables us to map on-chain redemptions with the data signed and collected off-chain, facilitating various physical events, primarily the shipping of a product associated with an NFT.
 
-Redemptions are non-destructive. When you redeem your NFT, it still exists after the redemption and you keep its ownership.
+Redemptions are non-destructive. When you redeem your NFT, it continues to exist after the redemption and you keep its ownership.
```
Comment (cosimopiu) [inline src/doc/fxhash/redeemables.md:15]: Redeeming your token does not destroy it, nor does it remove the token from your wallet or ownership. After redeeming your token, it remains collectible, salable, and transferable in its own right.

File: src/doc/fxhash/redeemables.md [hunk 3]
```diff
@@ -8,15 +8,15 @@ description: "Redeemables provide a framework to connect NFTs with their physica
...
 
-# Is the feature opened ?
```
Comment (cosimopiu) [inline src/doc/fxhash/redeemables.md:17]: # Is the feature open to everyone?

File: src/doc/fxhash/redeemables.md [hunk 4]
```diff
@@ -8,15 +8,15 @@ description: "Redeemables provide a framework to connect NFTs with their physica
...
+# Is the feature open ?
 
-Redeemables are rather complex, and for now can only be created from our admin dashboard. Before releasing the feature to the public, we want to carefully test it and make sure we cover the basic use-cases. We are opened to receiving requests if you want to experiment with redeemables.
+Redeemables are rather complex, and for now, can only be created from our internal admin dashboard. Before releasing the feature to the public, we want to carefully test it and ensure we cover the basic use cases. We are open to receiving requests if you want to experiment with redeemables.
 
 # How can I request access to have my project enabled through redeemables ?
```
Comment (cosimopiu) [inline src/doc/fxhash/redeemables.md:21]: # How can I request access to have my project enabled through redeemables?

File: src/doc/fxhash/redeemables.md [hunk 5]
```diff
@@ -8,15 +8,15 @@ description: "Redeemables provide a framework to connect NFTs with their physica
...
+Redeemables provide a set of modules allowing the platform to request arbitrary data. This data is then signed and is associated with a redeem event when it occurs on-chain. It enables us to map on-chain redemptions with the data signed and collected off-chain, facilitating various physical events, primarily the shipping of a product associated with an NFT.
 
-Redemptions are non-destructive. When you redeem your NFT, it still exists after the redemption and you keep its ownership.
+Redemptions are non-destructive. When you redeem your NFT, it continues to exist after the redemption and you keep its ownership.
 
-# Is the feature opened ?
+# Is the feature open ?
 
-Redeemables are rather complex, and for now can only be created from our admin dashboard. Before releasing the feature to the public, we want to carefully test it and make sure we cover the basic use-cases. We are opened to receiving requests if you want to experiment with redeemables.
```
Comment (cosimopiu) [inline src/doc/fxhash/redeemables.md:19]: Redeemables are rather complex and, for now, can only be created via our internal admin dashboard. Before releasing the feature to the public, we want to test it carefully and ensure we cover the basic use cases. We are open to receiving requests if you want to experiment with redeemables.

**Review by cosimopiu — CHANGES_REQUESTED** [review]

**Review by ciphrd — APPROVED** [review]

**Review by cosimopiu — CHANGES_REQUESTED** [review]
Minor typo fix

**Pull request merged**